frontend/src/Editor/LeftSidebar/SidebarPageSelector/PageHandler.jsx

```diff
@@ -91,6 +97,12 @@ export const PageHandler = ({
       case 'unhide-page':
         unHidePage(page.id);
         break;
+      case 'disable-page':
+        disableEnablePage({ pageId: page.id, isDisabled: true });
+        break;
+      case 'enable-page':
+        disableEnablePage({ pageId: page.id });
```

Comment:
Skipping `isDisabled: false` isn't good as it requires fellow dev to read code to understand. A bit unpredictable about handling.